collect_app/src/main/java/org/odk/collect/android/formmanagement/download/ServerFormDownloader.java

```diff
@@ -75,10 +75,12 @@ public void downloadForm(ServerFormDetails form, @Nullable ProgressReporter prog
 
         File tempDir = new File(cacheDir, "download-" + UUID.randomUUID().toString());
         tempDir.mkdirs();
+        String tempMediaPath = new File(tempDir, "media").getAbsolutePath();
```

Comment:
Why do we need the changes here? I think you added something and then reverted that change, but not entirely.